premerge/premerge_resources/main.tf

```diff
@@ -111,6 +129,60 @@ resource "helm_release" "github_actions_runner_set_windows" {
   ]
 }
 
+resource "helm_release" "github_actions_runner_set_libcxx" {
+  name       = "llvm-premerge-libcxx-runners"
+  namespace  = "llvm-premerge-libcxx-runners"
+  repository = "oci://ghcr.io/actions/actions-runner-controller-charts"
+  version    = "0.11.0"
+  chart      = "gha-runner-scale-set"
+
+  values = [
+    "${templatefile("libcxx_runners_values.yaml", { runner_group_name : var.runner_group_name, runner_image : var.libcxx_runner_image })}"
+  ]
+
+  depends_on = [
+    kubernetes_namespace.llvm_premerge_libcxx_runners,
+    helm_release.github_actions_runner_controller,
+    kubernetes_secret.linux_github_pat,
```

Comment:
My earlier comment probably was pretty vague about the secrets (or I was misreading things), but we do need kubernetes secrets for each of the new namespaces as secrets in k8s are namespaced. You can essentially just copy and paste the definition of `kubernetes_secret.linux_github_pat`, adjusting the namespaces as necessary.